tests/js/spec/views/organizationIntegrations/sentryAppDetailedView.spec.jsx

```diff
@@ -0,0 +1,361 @@
+    projects: [
+      {isMember: true, isBookmarked: true},
+      {isMember: true, slug: 'new-project', id: 3},
+    ],
+    organization: {
+      features: ['events', 'internal-catchall'],
+    },
+    router: {
+      location: {
+        pathname: '/organizations/org-slug/events/',
+        query: {},
+      },
+    },
+  });
+
+  describe('Published Sentry App', function() {
+    beforeEach(() => {
+      Client.clearMockResponses();
+
+      mockResponse([
+        [
+          '/sentry-apps/clickup/',
+          {
+            status: 'published',
+            scopes: [],
+            isAlertable: false,
+            clientSecret:
+              '193583e573d14d61832de96a9efc32ceb64e59a494284f58b50328a656420a55',
+            overview: null,
+            verifyInstall: false,
+            owner: {id: 1, slug: 'sentry'},
+            slug: 'clickup',
+            name: 'ClickUp',
+            uuid: '5d547ecb-7eb8-4ed2-853b-40256177d526',
+            author: 'Nisanthan',
+            webhookUrl: 'http://localhost:7000',
+            clientId: 'c215db1accc040919e0b0dce058e0ecf4ea062bb82174d70aee8eba62351be24',
+            redirectUrl: null,
+            allowedOrigins: [],
+            events: [],
+            schema: {},
+          },
+        ],
+        [
+          '/sentry-apps/clickup/features/',
+          [
+            {
+              featureGate: 'integrations-api',
+              description:
+                'ClickUp can **utilize the Sentry API** to pull data or update resources in Sentry (with permissions granted, of course).',
+            },
+          ],
+        ],
+        [`/organizations/${org.slug}/sentry-app-installations/`, []],
+        [
+          `/organizations/${org.slug}/sentry-app-installations/`,
+          {
+            status: 'installed',
+            organization: {slug: `${org.slug}`},
+            app: {uuid: '5d547ecb-7eb8-4ed2-853b-40256177d526', slug: 'clickup'},
+            code: '1dc8b0a28b7f45959d01bbc99d9bd568',
+            uuid: '687323fd-9fa4-4f8f-9bee-ca0089224b3e',
+          },
+          'POST',
+        ],
+      ]);
+
+      wrapper = mountWithTheme(
+        <SentryAppDetailedView
+          params={{integrationSlug: 'clickup', orgId: org.slug}}
+          location={{query: {}}}
+        />,
+        routerContext
+      );
+    });
+    it('shows the Integration name and install status', async function() {
+      expect(wrapper.find('Name').props().children).toEqual('ClickUp');
+      expect(wrapper.find('IntegrationStatus').props().status).toEqual('Not Installed');
+    });
+    it('shows the Accept & Install button', async function() {
+      expect(wrapper.find('InstallButton').props().disabled).toEqual(false);
+      expect(wrapper.find('InstallButton').props().children).toEqual('Accept & Install');
+    });
+
+    it('onClick: installs app', async function() {
+      wrapper.find('InstallButton').simulate('click');
+      await tick();
+      wrapper.update();
```

Comment:
@NisanthanNanthakumar You should check that the installation endpoint was actually called